docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -0,0 +1,181 @@
+[[open-telemetry-elastic]]
+== OpenTelemetry integration
+
+:ot-spec:       https://github.com/open-telemetry/opentelemetry-specification/blob/master/README.md
+:ot-repo:       https://github.com/open-telemetry/opentelemetry-collector
+:ot-pipelines:  {ot-repo}/blob/master/docs/pipelines.md
+:ot-extension:  {ot-repo}/blob/master/extension/README.md
+:ot-scaling:    {ot-repo}/blob/master/docs/performance.md
+
+:ot-collector:  https://opentelemetry.io/docs/collector/about/
+:ot-dockerhub:  https://hub.docker.com/r/otel/opentelemetry-collector-contrib-dev
+
+Elastic's OpenTelemetry integration allows you to reuse your existing OpenTelemetry
+instrumentation to quickly analyze distributed traces and metrics with the Elastic Stack.
+
+[float]
+[[what-is-otel]]
+=== What is OpenTelemetry?
+
+> OpenTelemetry is a set of APIs, SDKs, tooling, and integrations that enable the creation and
+management of telemetry data. It formed through a merger of the OpenTracing and OpenCensus projects.
+
+OpenTelemetry is an open-source project that provides the components necessary to observe your applications and services.
+If you're unfamiliar with the project, see the {ot-spec}[spec] for more information.
+
+[float]
+[[open-telemetry-elastic-exporter]]
+=== Elastic exporter
+
+Elastic's integration is designed to drop into your current OpenTelemetry setup.
+We've done this by extending the default OpenTelemetry collector and adding an Elastic exporter.
+This exporter translates the OpenTelemetry trace data collected from your services to Elastic's protocol,
+before sending the data to the Elastic Stack.
+By extending the OpenTelemetry collector,
+no changes are needed in your instrumented services in order to begin using the Elastic Stack.
+
+*************
```

Comment:
I guess this will be replaced with an image at some point? It doesn't render well in the browser.